docs/grammar/BallerinaLexer.g4

```diff
@@ -71,6 +71,8 @@ BIND        : 'bind' ;
 IN          : 'in' ;
 LOCK        : 'lock' ;
 
+DOCUMENTATION  : 'documentation';
```

Comment:
This is a top-level construct like function, services, etc. So this need to be grouped with them.

Reply:
+1